src/include/processor/operator/persistent/reader.h

```diff
@@ -40,13 +50,18 @@ class Reader : public PhysicalOperator {
     void getNextNodeGroupInParallel();
 
 private:
-    ReaderInfo readerInfo;
+    std::unique_ptr<ReaderInfo> info;
     std::shared_ptr<storage::ReaderSharedState> sharedState;
-    std::unique_ptr<common::DataChunk> dataChunkToRead;
+
     storage::LeftArrowArrays leftArrowArrays;
 
+    std::unique_ptr<common::DataChunk> dataChunk = nullptr;
```

Comment:
Maybe move all initializations to constructor. Right now, i see `readFuncData` is initialized in the constructor, while others are here. Let's unify them.